homework-classes/RepoView.js

```diff
@@ -18,9 +18,40 @@
      * Renders the repository details.
      * @param {Object} repo A repository object.
      */
+
     render(repo) {
-      // TODO: replace this comment and the console.log with your own code
       console.log('RepoView', repo);
+      const repoContainer = document.querySelector('.repo-container');
+      const select = document.querySelector('.repo-select');
+
+      function addRow(table, title, repoText) {
+        const tr = createAndAppend('tr', table);
+        createAndAppend('th', tr, {
+          text: title,
+        });
+        createAndAppend('td', tr, { text: repoText });
+        return tr;
+      }
+
+      function renderRepoDetails(repo, parent) {
```

Comment:
Besides my earlier point about embedded functions, defining this code as a function is unneeded as you call it immediately in line 50.